mobify/platform-scaffold
# Use SDK version of the modal branch

This PR removes the modal branch code from the scaffold and instead hooks in the SDK version included in the linked PR.

 **JIRA**: Preparatory for moving the integration manager into the SDK
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/569

## Changes
- remove the modal branch code
- use the SDK code

## How to test-drive this PR
- link in the associated SDK branch
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Make sure modals still open and close
